packages/editor-ui/src/plugins/codemirror/completions/nativesAutocompleteDocs/luxon.instance.docs.ts

```diff
@@ -210,17 +288,23 @@ export const luxonInstanceDocs: Required<NativeDoc> = {
 		year: {
 			doc: {
 				name: 'year',
+				description: i18n.baseText('codeNodeEditor.completer.luxon.instanceMethods.year'),
 				section: 'query',
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimeyear',
 				returnType: 'number',
+				examples: [{ example: "dt = '2024-03-30T18:49'.toDateTime()\ndt.year", evaluated: '2024' }],
 			},
 		},
 		zone: {
 			doc: {
 				name: 'zone',
+				description: i18n.baseText('codeNodeEditor.completer.luxon.instanceMethods.zone'),
 				section: 'query',
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimezone',
 				returnType: 'Zone',
+				examples: [
+					{ example: '$now.zone', evaluated: '{ zoneName: "Europe/Berlin", valid: true }' },
```

Comment:
I know we're just wrapping Luxon's API, but I really wonder if a user expects this object to be the evaluated result rather than the timezone string.

Reply:
We could add an extension method that returns just the string, but not sure if it's worth it